### PR TITLE
chore: add calling  assets(AR-1250)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1250" title="AR-1250" target="_blank">SubtaskAR-1250</a>  create UI
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

New feature

### Solutions

Adding calling assets

#### How to Test

Not needed

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-483/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3029859)(3029859)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3029859)(3029859)-dev-release.apk)

File: app/src/main/res/drawable/ic_speaker_off.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (saleniuk) [inline app/src/main/res/drawable/ic_speaker_off.xml:1]: There are `camera_off` and `camera_on` but only `speaker_off` - don't we need also `speaker_on`? 😄

Comment (ohassine) [reply app/src/main/res/drawable/ic_speaker_off.xml:1]: Right, the icon is not ready yet in figma.. I will add it later once it's available

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-483/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3035814)(3035814)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3035814)(3035814)-dev-release.apk)

**Review by saleniuk — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-483/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3045112)(3045112)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3045112)(3045112)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-483/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3045603)(3045603)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3045603)(3045603)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-483/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3045800)(3045800)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3045800)(3045800)-dev-release.apk)

**Pull request merged**